public/app/features/canvas/elements/cloud.tsx

```diff
@@ -184,6 +184,24 @@ export const cloudItem: CanvasElementItem = {
         },
       });
   },
+
+  customConnectionAnchors: [
+    { x: -0.58, y: 0.63 }, // Top Left
```

Comment:
Might be better to have property names? Seems a little cleaner and could be a good follow-up